add-logo-to-sidebar.patch

This patch enables the html_logo option

diff --git a/guzzle_sphinx_theme/guzzle_sphinx_theme/logo-text.html b/guzzle_sphinx_theme/guzzle_sphinx_theme/logo-text.html
index e52500e..e7c949c 100644
--- a/guzzle_sphinx_theme/guzzle_sphinx_theme/logo-text.html
+++ b/guzzle_sphinx_theme/guzzle_sphinx_theme/logo-text.html
@@ -1 +1,8 @@
-<a href="{{ homepage() }}" class="text-logo">{{ theme_project_nav_name or shorttitle }}</a>
+{% if logo %}
+  <a href="{{ homepage() }}" class="logo">
+  {# Not strictly valid HTML, but it's the only way to display/scale it properly, without weird scripting or heaps of work #}
+  <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" />
+  </a>
+{% else %}
+  <a href="{{ homepage() }}" class="text-logo">{{ theme_project_nav_name or shorttitle }}</a>
+{% endif %}
diff --git a/guzzle_sphinx_theme/guzzle_sphinx_theme/static/guzzle.css_t b/guzzle_sphinx_theme/guzzle_sphinx_theme/static/guzzle.css_t
index 4fdfca1..5379195 100644
--- a/guzzle_sphinx_theme/guzzle_sphinx_theme/static/guzzle.css_t
+++ b/guzzle_sphinx_theme/guzzle_sphinx_theme/static/guzzle.css_t
@@ -326,6 +326,16 @@ div.sphinxsidebar input {
   color: #fff;
 }
 
+.logo {
+    display:block;
+    margin:0 auto;
+    height:auto;
+    width:auto;
+    border-radius:0;
+    max-width:100%;
+    background:transparent;
+}
+
 /* Left-nav search box
 -------------------------------------------------- */
 
